server/app.js

```diff
@@ -45,6 +49,8 @@ app.use((req, res, next) => {
 app.use("/auth", authRouter);
```

Comment:
Just wondering why you removed the logger!, I mean the morgan, did you move it to another place or something

Reply:
do we need logger in app.js? looks like they are not using so I delete

Reply:
In the way I see it, loggers are good, for the local development `morgan` is okay, for deployment I would go with `winston`,
On the same note ask @moffatethan  and see if he can allow creating a setup file so that we can move the configuration to that file and reduce the load from the `app.js`

Reply:
I still see the logger being used here 👍🏻